benchmark/utils.py

```diff
@@ -0,0 +1,58 @@
+import os
+import subprocess
+import sys
+import time
+import torch
+
+model_dir = 'models/'
+install_file = 'install.py'
+hubconf_file = 'hubconf.py'
+
+
+def _install_deps(model_path):
+    if os.path.exists(os.path.join(model_path, install_file)):
+        subprocess.check_call([sys.executable, install_file], cwd=model_path)
+    else:
+        print('skip installing deps and preprocessing since no install.py is found in {}.'.format(model_path))
+
+
+def list_models():
+    return os.listdir(model_dir)
+
+
+def setup():
+    models = list_models()
+    for model in models:
+        _install_deps(model_dir + model)
+
+
+def run_models():
+    models = list_models()
+    for model in models:
+        cwd = os.getcwd()
+        model_path = os.path.join(cwd, model_dir, model)
+        sys.path.insert(0, model_path)
+        os.chdir(model_path)
+        hub_module = torch.hub.import_module(hubconf_file, hubconf_file)
+        Model = getattr(hub_module, 'Model', None)
+        if Model:
+            m = Model()
```

Comment:
this seems like an extra step since the Model class takes no arguments. Two options:

1. the constructor to Module takes the configuration flags: `jit`, `device`, etc. And we drop the `set_x` functions.
2. We remove the constructor and have `hub_module.model` be an instance of the `Model` class.

I think I like option 1 better since it allows the Model constructor to perform the setup all in one place.